### PR TITLE
Edit caption to more descriptive

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @ericalloyd
- [ ] @BrandyMartin
- [ ] @imrandyk
- [ ] @jasonthornhill
- [ ] @jcoz1968
- [ ] @jessicakersey
- [ ] @jlamons
- [ ] @juneomes
- [ ] @matt9388
- [ ] @mrodriguez-omes
- [ ] @tfinch00

### PR DESCRIPTION
**Pull request by juneomes**
Fixes #2

### CODE REVIEW

File: _slides/09-juneomes.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+A beatiful day for coderCAT!
```
Comment (matt9388) [inline _slides/09-juneomes.md:6]: Nice!

Comment (BrandyMartin) [inline _slides/09-juneomes.md:6]: Yes it is!

Comment (matt9388) [inline _slides/09-juneomes.md:6]:
```suggestion
A beatiful day for coderCAT!
```

Comment (matt9388) [inline _slides/09-juneomes.md:6]:
```suggestion
A beatiful day for coderCAT!
```

**Review by matt9388 — APPROVED** [review]

**Review by BrandyMartin — APPROVED** [review]

**Pull request merged**